src/Appliance_Item_Relation.php

```diff
@@ -275,7 +275,7 @@ public static function getListJSForApplianceItem(
                var appliances_items_id = $(this).data('appliances-items-id');
 
                glpi_html_dialog({
-                  title: _x('button', "Add an item"),
+                  title: _sx('button', "Add an item"),
```

Comment:
The escaping should be done in the `glpi_html_dialog` function itself.
```suggestion
                  title: _x('button', "Add an item"),
```